core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -447,6 +448,38 @@ private function addMailerNode(): NodeDefinition
         ;
     }
 
+    private function addBackendNode(): NodeDefinition
+    {
+        return (new TreeBuilder('backend'))
+            ->getRootNode()
+            ->addDefaultsIfNotSet()
+            ->children()
+                ->arrayNode('attributes')
+                    ->info('Adds data attributes to the body of the back end.')
+                    ->example(['app-name' => 'My App', 'app-version' => '1.2.3'])
+                    ->normalizeKeys(false)
+                    ->useAttributeAsKey('name')
+                    ->scalarPrototype()->end()
```

Comment:
Should validate the keys do not contain any characters that are not allowed in `data-` attributes (e.g. whitespace...).

Reply:
Oh right nice idea with validation at this point!